test/components/views/settings/__snapshots__/UiFeatureSettingWrapper-test.tsx.snap

```diff
@@ -1,11 +1,74 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`<UiFeatureSettingWrapper> renders children when setting is truthy 1`] = `
-<UiFeatureSettingWrapper
-  uiFeature="UIFeature.feedback"
->
-  <div>
-    test
-  </div>
-</UiFeatureSettingWrapper>
+{
+  "asFragment": [Function],
```

Comment:
Snapshots need updating to be passed `asFragment()` instead of the whole wrapper

Reply:
TIL 👍